actionpack/lib/action_controller/api.rb

```diff
@@ -0,0 +1,157 @@
+require 'action_view'
+require 'action_controller'
+require 'action_controller/log_subscriber'
+
+module ActionController
+  # API Controller is a lightweight version of <tt>ActionController::Base</tt>,
+  # created for applications that don't require all functionality that a complete
+  # \Rails controller provides, allowing you to create faster controllers for
```

Comment:
Loading less is inherently faster than loading more. The statement is correct. Weather or not it's relevant to the real world is debatable.